features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularContext.cpp

```diff
@@ -69,11 +69,20 @@ ControlPlane_netif *QUECTEL_BG96_CellularContext::get_cp_netif()
 
 nsapi_error_t QUECTEL_BG96_CellularContext::do_user_authentication()
```

Comment:
`AT_CellularContext::get_context` still uses `AT+CGDCONT` so that should be overridden as well.

Reply:
`AT_CellularContext::get_context` is using `AT+CGDCONT` only for reading. No need for QICSGP. If you think set_new_context(), that has to be done still by CGDCONT.QICSGP can only congfigure a context created by CGDCONT.

Reply:
QICSGP must be called to configure APN. If don't want to override `get_context` then should allow empty `_pwd` and `_uname` here.

Reply:
Isn't [this](https://github.com/ARMmbed/mbed-os/blob/eb78ba662493b689b5422535d1dc861c4237dd63/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularContext.cpp#L82)  allowing empty `_pwd `and `_uname`?